networking/network_security/network_policy/multitenant-network-policy.adoc

```diff
@@ -28,5 +28,5 @@ ifndef::openshift-rosa,openshift-dedicated[]
 [id="multitenant-network-policy-additional-resources"]
 == Additional resources
 
-* xref:../../../networking/openshift_sdn/about-openshift-sdn.adoc#nw-openshift-sdn-modes_about-openshift-sdn[OpenShift SDN network isolation modes]
+* xref:../../../networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc#about-ovn-kubernetes[About the OVN-Kubernetes network plugin]
```

Comment:
```suggestion
* xref:../../../networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc#about-ovn-kubernetes[About the OVN-Kubernetes network plugin]
```
Perhaps note this for later because it is in a module name..."plugin".